.github/workflows/scripts/check_translation_usage.py

```
@@ -0,0 +1,331 @@
+    if isinstance(obj, dict):
+        for k, v in obj.items():
+            new_prefix = f"{prefix}.{k}" if prefix else k
+            keys.update(flatten_json(v, new_prefix))
+    elif prefix:
+        keys.add(prefix)
+    return keys
+
+
+def load_locales(
+    locales_dir: str,
+    verbose: bool = False,
```

Ruff (0.14.8)

Boolean-typed positional argument in function definition

(FBT001)